### DISCUSSION TITLE
Product Variant or Question with Age as base of Birthdate

### DISCUSSION THREAD

**Discussion by kempert** [pretix/pretix#3760] [Q&A]
Hallo, we want to show our Check-In users the Age of the Participants. Is there any way to calculate the Age of the partictpants by Base of the Brithday, and the Start of the event and write this in the hidden Question "Age" in the Backend? And set the right Product Variant for age range. 

Is there any way to do that? Thanks :)

**Comment by raphaelm** [comment]
No, this is not possible
